### PR TITLE
Import kubernetes updates

### PR DESCRIPTION
**Pull request by serathius**
examples directory state from https://github.com/kubernetes/kubernetes/commit/b79fe1073017907a59a869413d659f74e90b0443

Process:
* start with kubernetes master
* remove all content and history for all directories beside `examples` directory using `git filter-branch --filter-subdirectory`
* remove all commits that overwrite README with references to this repo
* find newest common commit in history and following it commit to move examples to staging directory
* rebase kubernetes repo for `examples` to this commit
* merge with this repo master

Strategy for merge:
* Remove all BUILD files
* Move new examples to staging directory
* Remove changes to main directory 
* Try to fix files based on git blame, and which links work which not

First stage for https://github.com/kubernetes/kubernetes/issues/60887

### CODE REVIEW

**Comment by serathius** [comment]
cc @ahmetb

**Comment by serathius** [comment]
What we want to do with historical contributors without CLA signed?

**Comment by serathius** [comment]
Looks like one of emails used by contributor was dissociated with Github account

**Comment by ahmetb** [comment]
We'll likely skip the CLA check for this particular PR.

**Comment by ahmetb** [comment]
/cc @idvoretskyi 
/cc @sebgoa 
for review help

File: staging/persistent-volume-provisioning/README.md [hunk 1]
```diff
@@ -95,6 +97,30 @@ parameters:
...
+  adminSecretName: storageos-secret
+```
+
+*  `pool`: The name of the StorageOS distributed capacity pool to provision the volume from.  Uses the `default` pool which is normally present if not specified.
+*  `description`: The description to assign to volumes that were created dynamically.  All volume descriptions will be the same for the storage class, but different storage classes can be used to allow descriptions for different use cases.  Defaults to `Kubernetes volume`.
+*  `fsType`: The default filesystem type to request.  Note that user-defined rules within StorageOS may override this value.  Defaults to `ext4`.
+*  `adminSecretNamespace`: The namespace where the API configuration secret is located. Required if adminSecretName set.
+*  `adminSecretName`: The name of the secret to use for obtaining the StorageOS API credentials.  If not specified, default values will be attempted.
+
+For a complete example refer to the ([StorageOS example](../../volumes/storageos/README.md))
```
Comment (ahmetb) [inline staging/persistent-volume-provisioning/README.md]: it seems like all the `../../volumes` references on this page are broken. these all should be  `../../staging/volumes`

**Comment by ahmetb** [comment]
This looks great to me, I wasn't able to find many issues other than a few broken links (which I think you 
can address by adding a new commit). Thanks for doing this work while preserving the commit history!
/lgtm
/hold

**Comment by idvoretskyi** [comment]
We need to check the CLA question.

@dankohn @caniszczyk - does it sound like an issue from the legal POV now?

**Comment by idvoretskyi** [comment]
@serathius @ahmetb I've been speaking to @caniszczyk - it's not allowed from the legal perspective to have the commits without the signed CLA.

**Comment by ahmetb** [comment]
@idvoretskyi To be clear, while creating the repo we did the same thing: We brought the commits that were in `kubernetes/kubernetes` to `kubernetes/examples`. Now this PR is doing it again to bring these two repos to sync.

@idvoretskyi / @caniszczyk  Please answer these following questions:

1. All commits in `kubernetes/kubernetes` repo has CLA signed, is this true?
2. If (1) is true, then we know that repos like `kubernetes/ingress-gce` were created from `kubernetes/ingress` by following the same process, which is copying the commit history. (I can probably find more examples of this practice.) What's the difference between those things and what we're doing here?
3. Is this CLA failure likely happening because some of the committers’ email addresses were removed from CLA database?
4. If (3) is true, they once allowed CNCF to have copyrights for this code. Now we're moving it from one CNCF property to another, within the Kubernetes project. What's the issue here?

**Comment by idvoretskyi** [comment]
@ahmetb apologies for the confusion. We've just double checked with the LF legal - we are able to migrate them in since the Google CLA was previously signed.

**Comment by ahmetb** [comment]
@idvoretskyi shall we flip the CLA label?

**Comment by idvoretskyi** [comment]
@ahmetb yes, please.

**Comment by ahmetb** [comment]
Switching labels manually didn't make the cla check happy, but now it lets me to merge manually.

**Pull request merged**